### DISCUSSION TITLE
Add Fathom as an analytics provider?

### DISCUSSION THREAD

**Discussion by SeanKilleen** [mmistakes/minimal-mistakes#3373] [Ideas]
Hi all,

Is there any support behind adding [Fathom](https://usefathom.com/) as an analytics provider? 

I'm happy to do the PR (have done similar PRs before for other things); I just want to make sure it's welcome first.

@mmistakes do you have an opinion on this?

**Comment by mmistakes** [comment]
I don't know if there's a big enough demand for it to bother adding a flag in `_config.yml`.
There's a custom include for anything the theme doesn't already cover that would fit this case.

https://mmistakes.github.io/minimal-mistakes/docs/configuration/#analytics
https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/analytics-providers/custom.html

**Reply by SeanKilleen** [reply]
Fair enough, and I am using the custom provider to accomplish it now.

I was more thinking that Fathom as a GA alternative that is privacy focused and doesn't even require tracking cookies is pretty darn good, and that the top-billing in something like minimal mistakes would bring more attention to it. So more of a "for the good of the community" type thing.

If there isn't an appetite for it, I'll continue using the custom provider. Thanks!
